osu.Game.Rulesets.Mania/Objects/Drawables/DrawableHoldNote.cs

```diff
@@ -252,6 +231,10 @@ protected override void CheckForJudgements(bool userTriggered, double timeOffset
                 });
             }
 
+            protected override void UpdateState(ArmedState state)
+            {

```

Comment:
I think this could probably do with a comment in the body just mentioning why we are inhibiting the base call here